src/app/screens/ConfirmPayment/index.jsx

```diff
@@ -58,8 +58,8 @@ class ConfirmPayment extends React.Component {
         <div className="p-6">
           <div className="mb-8">
             <PaymentSummary
-              amount={this.props.invoice?.valueSat}
-              description={this.props.invoice?.desc}
+              amount={this.props.invoice?.mtokens}
```

Comment:
```suggestion
              amount={this.props.invoice?.tokens}
```

I think this should be tokens. (not m(illi)tokens)

lightning supports sub-satoshi amounts and some amouts are represented as milli-satoshis. 

Currently we have only been using satoshi. But we should probably store the smallest possible amount and thus always use milli-satoshi and convert to sats (or other denominations) in the UI?